source/SpriteSet.h

```diff
@@ -34,10 +34,6 @@ class SpriteSet {
 	// Inspect the sprite map and warn if some images contain no data.
 	static void CheckReferences();
 
-
-private:
-	// Only SpriteQueue is allowed to modify the sprites.
```

Comment:
so now everybody is allowed to modify sprites? not sure if I am fan of that

Reply:
Yup. However, the next step in the refactoring process is do make this a normal class.

Reply:
in that case I understand

Reply:
https://github.com/endless-sky/endless-sky/pull/6892

Reply:
ig that would be important for #9868